test/old/testdir/test_search_stat.vim

```diff
@@ -75,53 +75,61 @@ func Test_search_stat()
   call assert_notmatch(pat .. stat, g:a)
   " n does not update search stat
   call assert_equal(
-    \ #{current: 50, exact_match: 1, total: 50, incomplete: 0, maxcount: 99},
+    \ #{current: 50, exact_match: 1, total: 50, incomplete: 0, maxcount: 999},
     \ searchcount(#{recompute: 0}))
   call assert_equal(
-    \ #{current: 2, exact_match: 1, total: 50, incomplete: 0, maxcount: 99},
+    \ #{current: 2, exact_match: 1, total: 50, incomplete: 0, maxcount: 999},
     \ searchcount(#{recompute: v:true}))
   set shortmess-=S
 
+  " Nvim: max search count is 999; append more lines to surpass it.
+  " Create an new undo block so we can revert the change later.
```

Comment:
```suggestion
  " Create a new undo block so we can revert the change later.
```

Reply:
me good at write englandish :troll::pencil2:

Reply:
cuz too good at write vimscript 🤗

Reply:
now time to fix the functionaltests :heart_eyes:

Reply:
Ah, the joy of hardcoded screen tests...